integration-tests/README.md

```
@@ -0,0 +1,81 @@
+# CLP integration tests
+
+This Python project provides end-to-end tests for CLP using the `pytest` framework.
+
+# Contributing
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)